frontend/components/notifications/management/NotificationsManagementDashboards.vue

```diff
@@ -222,6 +222,9 @@ function getTypeIcon(type: DashboardType) {
 const handleDelete = (payload: Parameters<typeof deleteDashboardNotifications>[0]) => {
   deleteDashboardNotifications(payload).then(() => refreshOverview())
 }
+const isDeleteButtonDisabled = (dashboard: WrappedRow) => {
+  return !dashboard.subscriptions?.length || dashboard.is_archived
+}
```

Comment:
easier to reason about:
```suggestion
const isDeleteButtonDisabled = (dashboard: WrappedRow) => {
  const hasSubscriptions = dashboard.subscriptions?.length
  return !hasSubscriptions || dashboard.is_archived
}
```

Reply:
Done